pasha/context.py

```diff
@@ -257,7 +454,7 @@ def array(self, shape, dtype=np.float64):
                         prot=mmap.PROT_READ | mmap.PROT_WRITE)
 
         return np.frombuffer(memoryview(buf)[:n_bytes],
-                             dtype=dtype).reshape(shape)
+                             dtype=dtype).reshape(shape, order=order)
```

Comment:
This is tying my brain in knots trying to work out if it does the right thing. I guess you've got enough tests that it would fail if it was wrong, but...

Reply:
As a matter of fact, I haven't included `ProcessContext` in the tests yet, but fixed it now. I think I'm testing every possible combination of arguments 🤓